okonomiyaki/runtimes/tests/test_runtime_info.py

```diff
@@ -39,6 +39,27 @@ def test_simple(self):
         self.assertEqual(runtime_info.name, name)
         self.assertEqual(runtime_info.executable, r_executable)
 
+    @unittest.skipIf(not sys.platform.startswith('win'), 'Test case unique to Windows')
```

Comment:
If we only test on windows while the changes in the code are generic it looks that we are not testing correctly.

Reply:
You are right. I should test this on all platforms.